ethereum-consensus/src/deneb/blob_sidecar.rs

```diff
@@ -27,6 +29,95 @@ pub struct BlobSidecar<
     pub kzg_commitment_inclusion_proof: Vector<Bytes32, KZG_COMMITMENT_INCLUSION_PROOF_DEPTH>,
 }
 
+pub fn verify_blob_sidecar_inclusion_proof<
+    const MAX_PROPOSER_SLASHINGS: usize,
+    const MAX_VALIDATORS_PER_COMMITTEE: usize,
+    const MAX_ATTESTER_SLASHINGS: usize,
+    const MAX_ATTESTATIONS: usize,
+    const MAX_DEPOSITS: usize,
+    const MAX_VOLUNTARY_EXITS: usize,
+    const SYNC_COMMITTEE_SIZE: usize,
+    const BYTES_PER_LOGS_BLOOM: usize,
+    const MAX_EXTRA_DATA_BYTES: usize,
+    const MAX_BYTES_PER_TRANSACTION: usize,
+    const MAX_TRANSACTIONS_PER_PAYLOAD: usize,
+    const MAX_WITHDRAWALS_PER_PAYLOAD: usize,
+    const MAX_BLS_TO_EXECUTION_CHANGES: usize,
+    const MAX_BLOB_COMMITMENTS_PER_BLOCK: usize,
+    const KZG_COMMITMENT_INCLUSION_PROOF_DEPTH: usize,
+>(
+    index: BlobIndex,
+    kzg_commitment: &mut KzgCommitment,
+    signed_block_header: &SignedBeaconBlockHeader,
+    kzg_commitment_inclusion_proof: &Vector<Bytes32, KZG_COMMITMENT_INCLUSION_PROOF_DEPTH>,
```

Comment:
ok, one last thing I think!

let's mirror [the spec](https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/p2p-interface.md#verify_blob_sidecar_inclusion_proof) and take a `BlobSidecar` as an input!